requirements.txt

```diff
@@ -7,6 +7,8 @@ importlib-resources==5.8.0 ; python_version=="3.7.*" or python_version=="3.8.*"
 Jinja2==3.1.2
 MarkupSafe==2.1.1
 Pygments==2.12.0
+pyobjc-core==8.5 ; sys_platform=="darwin"
+pyobjc-framework-Cocoa==8.5 ; sys_platform=="darwin"
```

Comment:
Could you please also:

- Edit `README.asciidoc` to point out the additional dependency on macOS
- Add it to `MODULE_INFO` in `qutebrowser/utils/version.py` so that it gets shown in `:version` (for now you can just add `objc` with `__version__` unconditionally, it will just show `objc: no` if unavailable). That code is refactored a bit in the `qt6-v2` branch to allow conditional entries, so once that's merged, I suppose we can change it to not display on non-macOS at all.
- Adjust `tests/unit/utils/test_version.py` accordingly - at least add it to `test_existing_attributes`, not sure off-hand if any other changes will be needed.